api_server/src/http_service.rs

```diff
@@ -795,6 +795,11 @@ mod tests {
         );
     }
 
+    #[test]
```

Comment:
Please move this file to a different commit, as the changes are unrelated to the `chrono` dependency removal.